Meshery-Adapters/consul-meshery-adapter/step1-foreground.sh

```diff
@@ -1,3 +1,6 @@
 #!/bin/bash
 
-/root/shell.sh
+while [ ! -e /root/shell.sh ];do
+    sleep 1s
+done
+/root/shell.sh
```

Comment:
This one is great!

There is just one problem, foreground will print all steps while executing, like

```
> while [ ! -e /root/shell.sh ];do
>  sleep 1s
```

Can we remove shell.sh and in foreground we can place `minikube start`  so that user know Minikube is starting
and in background we can place `cd $HOME/.kube && kubectl config view --minify --flatten > config.yaml && mv config.yaml config`

Before jumping to  this one lets get views from @leecalcote

Reply:
@suhaniii's proposal seems to capture a good balance of letting the user know that the system is making progress, while hiding some of the more detailed particulars to avoid inundating users. Sounds good. 

Having said that, I'm curious as to whether we can use the "minikube-running" image or another image with K8s already running), so that minikube start isn't necessary. Is this possible?

Reply:
I have tried `minikube-running` myself, it's not there anymore.

this one is Minikube scenario on katacoda, we have to start Minikube 
 https://www.katacoda.com/scenario-examples/courses/environment-usages/minikube
https://www.katacoda.com/courses/kubernetes/launch-single-node-cluster

And the k8s with two nodes
https://www.katacoda.com/courses/kubernetes/playground

In both scenario we have to start the k8s

@devansh42 If you able to find image where we can avoid start then we can update that.

I guess there is an option for creating custom images if we have subscription?

Reply:
Oooh, I see. Hmpf. Thanks for investigating this @suhaniii. So these images have Kubernetes packages installed, but not running. 

@devansh42 have you seen another available image with Kubernetes running already?